### ISSUE TITLE
[FR/bug] Documents opened for the first time are completely zoomed-out

### ISSUE THREAD

**Issue by Elaws** [ahrm/sioyek#162]
Hello,

I've noticed a weird behavior where documents opened for the first time are completely zoomed out : 

![image](https://user-images.githubusercontent.com/52013479/156860871-647cc2f1-5070-4ac9-b713-ba6a03703811.png)

We could add `fit_to_page_width` or `fit_to_page_width_smart` startup command to the `prefs_user_config`, but then the page will appear completely zoomed in : 

![image](https://user-images.githubusercontent.com/52013479/156861053-72c43d11-be28-4a1f-86cf-739cd8e0c88b.png)

Is there any way to change the behavior so that documents opened for the first time would appear like that : 

![image](https://user-images.githubusercontent.com/52013479/156861152-f58fd84a-03ce-4952-9f1d-5f4a4daf4160.png)

Thanks !

**Comment by ahrm** [comment]
I was not able to reproduce your issue with the page being zoomed out. How are you opening the file? (are you using command line or are you clicking on the file? If you are using command line what command line options are you using?).
As for the page being too zoomed in, I added a `fit_to_page_width_ratio` command in cffcdcb8381a432fd0c81c99d4499a50ba3324d2. You can configure it in `prefs.config` like so:
```
fit_to_page_width_ratio 0.75
```
Now when you execute the `fit_to_page_width_ratio` command, we just use 75% of the screen width.

**Comment by Elaws** [comment]
Hello @ahrm : thank you for the `fit_to_page_width_ratio`.

I know exactly how to reproduce the issue !

Prerequisites :

- Document is opened by clicking on the file.
- You must have added `single_main_window_size` to your *prefs_user.config*.
- Make sure it's a document you never opened yet in Sioyek !
- (Optional : Delete your *shared.db* to make sure you are opening the document for the first time.)

The issue happens for the following values of `single_main_window_size` : 

- `single_main_window_size X Y` where `X = 1920` and `Y <= 1061`.

(My screen is 1920x1080).

**Comment by Elaws** [comment]
Mmm... By removing :

```c++
main_window->showMaximized(); // line 2963 of main_widget.cpp
```

The problem does not reproduce... But it's not a possible solution because when we remove this, the window does not fit properly to the screen. Not sure what is wrong with this `showMaximized` function...

**Comment by ahrm** [comment]
Should be fixed in 3a8282ea9d6ee5b1212393f886dc99d4a80194fd.

**Comment by Elaws** [comment]
Thanks a lot ! The problem is solved !
